src/extensions/snapshot/dedent.ts

```diff
@@ -81,8 +72,10 @@ module.exports = (text) => {
 
     const resultLines = []
     for (let i = 0; i < lines.length; i++) {
-        resultLines.push(lines[i].substring(skipLength))
+        resultLines.push(lines[i]?.substring(skipLength))
     }
 
     return resultLines.join('\n')
```

Comment:
return lines.map(line => line.substring(skipLength)).join('\n')

Reply:
yes, thank you for the suggestion!

Reply:
Thanks